SH4RE/SH4RE/Account/ReviewsView.swift

```diff
@@ -8,11 +8,31 @@
 import SwiftUI
 
 struct ReviewsView: View {
+    @State var allReviews = [Review]()
+    
     var body: some View {
-        ZStack {
+        ZStack(alignment: .top) {
             Color.backgroundGrey.ignoresSafeArea()
+            
+            VStack(alignment: .leading) {
+                Text("Reviews (\(allReviews.count))")
+                    .font(.headline)
+                    .padding()
+                
+                ForEach(allReviews) { review in
+                    ReviewView(reviewName: review.name, reviewRating: review.rating as Float, reviewDescription: review.description, reviewUID: review.uid)
+                }
+            }
+            .frame(maxWidth: .infinity, alignment: .leading)
         }
         .navigationTitle("Reviews")
+        .onAppear() {
+            getUserReviews(uid: getCurrentUserUid(), completion: { reviews in
```

Comment:
Can you use getUserRating to remove the placeholder in AccountView.swift and use getUserReviews to show all reviews in ProfileView.swift, similar to this. Both files have a variable `private var review` that has the placeholder